ansible-st2-local/roles/arteria_core/defaults/main.yml

```diff
@@ -1,11 +1,3 @@
 ---
-
-arteria_user: arteria
-arteria_uid: 4242
-arteria_group: arteria
-arteria_gid: 4242
-
-# Default to "prod". If set to "staging" some roles might have a slightly different
-# behavior, enabling debug mode and allowing extra integration tests capabilities.
-arteria_environment: prod
+arteria_disable_iptables: true
```

Comment:
Wasn't this removed by @johandahlberg in an earlier commit? Should it go back in or is this a merge conflict issue?